src/crepe/api/SceneManager.h

```diff
@@ -35,8 +42,11 @@ class SceneManager {
 	void load_next_scene();
 
 private:
+	//! Vector of scenes
```

Comment:
Ik kan al zien aan het type dat het een vector van scènes is, maar begrijp nog niet welke scènes dit zijn. Misschien toevoegen dat dit specifiek de lijst is van scènes die toegevoegd zijn door `add_scene`?